src/main/java/javaClasses/homework_4/davit_balabekyan/staff/Work.java

```diff
@@ -0,0 +1,19 @@
+package javaClasses.homework_4.davit_balabekyan.staff;
+
+public class Work {
+    public static void main(String[] args) {
+        Developer developer = new Developer();
+        developer.setName("Bill");
```

Comment:
As I mentioned above please create a constructor which will create your object by providing all required fields. In that case you can create your object with calling just 1 method instead of 3.

Reply:
Done)